From a52eb592ea4a6730a44fe323155989e35ca89c1c Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 8 Mar 2024 07:57:41 +0000
Subject: [PATCH 4/6] WIP: ARM: dts: add support for Minix Neo U9-H

Import device tree files for the Minix Neo U9-H from Linux 6.7.y and include
the minimal u-boot.dtsi file for GXM boards.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm/dts/meson-gxm-minix-neo-u9h-u-boot.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 arch/arm/dts/meson-gxm-minix-neo-u9h-u-boot.dtsi

diff --git a/arch/arm/dts/meson-gxm-minix-neo-u9h-u-boot.dtsi b/arch/arm/dts/meson-gxm-minix-neo-u9h-u-boot.dtsi
new file mode 100644
index 00000000000..39270ea71c8
--- /dev/null
+++ b/arch/arm/dts/meson-gxm-minix-neo-u9h-u-boot.dtsi
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2019 BayLibre, SAS.
+ * Author: Neil Armstrong <narmstrong@baylibre.com>
+ */
+
+#include "meson-gxl-u-boot.dtsi"
-- 
2.43.0

